build.yaml

```diff
@@ -90,24 +90,52 @@ common_cli:
     #
```

Comment:
its the shebang, which is still the first like of the script.
not sure what you meant here